Neos.Flow/Classes/I18n/Translator.php

```diff
@@ -119,9 +119,11 @@ public function injectPluralsReader(Cldr\Reader\PluralsReader $pluralsReader)
      * @param string $sourceName Name of file with translations, base path is $packageKey/Resources/Private/Locale/Translations/
      * @param string $packageKey Key of the package containing the source file
      * @return string Translated $originalLabel or $originalLabel itself on failure
+     * @throws Exception\IndexOutOfBoundsException
+     * @throws Exception\InvalidFormatPlaceholderException
      * @api
      */
-    public function translateByOriginalLabel($originalLabel, array $arguments = [], $quantity = null, Locale $locale = null, $sourceName = 'Main', $packageKey = 'Neos.Flow')
+    public function translateByOriginalLabel($originalLabel, array $arguments = [], $quantity = null, Locale $locale = null, string $sourceName = 'Main', string $packageKey = 'Neos.Flow')
```

Comment:
```suggestion
    public function translateByOriginalLabel(string $originalLabel, array $arguments = [], $quantity = null, Locale $locale = null, string $sourceName = 'Main', string $packageKey = 'Neos.Flow')
```

Reply:
Tried that, but there are tests, that fill it with null. So i reverted it again in https://github.com/neos/flow-development-collection/pull/1772/commits/c5d938b4e493c6c706226a41ed56c394e1a63419 and focused on getting it green.